crates/avalanche-types/src/proto/pb/sync.rs

```diff
@@ -11,133 +12,785 @@ pub mod request {
+    0x0a, 0x04, 0x04, 0x14, 0x02, 0x00, 0x12, 0x04, 0x97, 0x01, 0x02, 0x1b, 0x0a, 0x0d, 0x0a, 0x05,
+    0x04, 0x14, 0x02, 0x00, 0x05, 0x12, 0x04, 0x97, 0x01, 0x02, 0x08, 0x0a, 0x0d, 0x0a, 0x05, 0x04,
+    0x14, 0x02, 0x00, 0x01, 0x12, 0x04, 0x97, 0x01, 0x09, 0x16, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x14,
+    0x02, 0x00, 0x03, 0x12, 0x04, 0x97, 0x01, 0x19, 0x1a, 0x0a, 0x0c, 0x0a, 0x04, 0x04, 0x14, 0x02,
+    0x01, 0x12, 0x04, 0x98, 0x01, 0x02, 0x12, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x14, 0x02, 0x01, 0x05,
+    0x12, 0x04, 0x98, 0x01, 0x02, 0x07, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x14, 0x02, 0x01, 0x01, 0x12,
+    0x04, 0x98, 0x01, 0x08, 0x0d, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x14, 0x02, 0x01, 0x03, 0x12, 0x04,
+    0x98, 0x01, 0x10, 0x11, 0x0a, 0x0c, 0x0a, 0x02, 0x04, 0x15, 0x12, 0x06, 0x9b, 0x01, 0x00, 0xa0,
+    0x01, 0x01, 0x0a, 0x0b, 0x0a, 0x03, 0x04, 0x15, 0x01, 0x12, 0x04, 0x9b, 0x01, 0x08, 0x12, 0x0a,
+    0x0c, 0x0a, 0x04, 0x04, 0x15, 0x02, 0x00, 0x12, 0x04, 0x9c, 0x01, 0x02, 0x12, 0x0a, 0x0d, 0x0a,
+    0x05, 0x04, 0x15, 0x02, 0x00, 0x05, 0x12, 0x04, 0x9c, 0x01, 0x02, 0x07, 0x0a, 0x0d, 0x0a, 0x05,
+    0x04, 0x15, 0x02, 0x00, 0x01, 0x12, 0x04, 0x9c, 0x01, 0x08, 0x0d, 0x0a, 0x0d, 0x0a, 0x05, 0x04,
+    0x15, 0x02, 0x00, 0x03, 0x12, 0x04, 0x9c, 0x01, 0x10, 0x11, 0x0a, 0x40, 0x0a, 0x04, 0x04, 0x15,
+    0x02, 0x01, 0x12, 0x04, 0x9f, 0x01, 0x02, 0x16, 0x1a, 0x32, 0x20, 0x49, 0x66, 0x20, 0x66, 0x61,
+    0x6c, 0x73, 0x65, 0x2c, 0x20, 0x74, 0x68, 0x69, 0x73, 0x20, 0x69, 0x73, 0x20, 0x4e, 0x6f, 0x6e,
+    0x65, 0x2e, 0x0a, 0x20, 0x4f, 0x74, 0x68, 0x65, 0x72, 0x77, 0x69, 0x73, 0x65, 0x20, 0x74, 0x68,
+    0x69, 0x73, 0x20, 0x69, 0x73, 0x20, 0x53, 0x6f, 0x6d, 0x65, 0x2e, 0x0a, 0x0a, 0x0d, 0x0a, 0x05,
+    0x04, 0x15, 0x02, 0x01, 0x05, 0x12, 0x04, 0x9f, 0x01, 0x02, 0x06, 0x0a, 0x0d, 0x0a, 0x05, 0x04,
+    0x15, 0x02, 0x01, 0x01, 0x12, 0x04, 0x9f, 0x01, 0x07, 0x11, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x15,
+    0x02, 0x01, 0x03, 0x12, 0x04, 0x9f, 0x01, 0x14, 0x15, 0x0a, 0x0c, 0x0a, 0x02, 0x04, 0x16, 0x12,
+    0x06, 0xa2, 0x01, 0x00, 0xa5, 0x01, 0x01, 0x0a, 0x0b, 0x0a, 0x03, 0x04, 0x16, 0x01, 0x12, 0x04,
+    0xa2, 0x01, 0x08, 0x10, 0x0a, 0x0c, 0x0a, 0x04, 0x04, 0x16, 0x02, 0x00, 0x12, 0x04, 0xa3, 0x01,
+    0x02, 0x10, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x16, 0x02, 0x00, 0x05, 0x12, 0x04, 0xa3, 0x01, 0x02,
+    0x07, 0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x16, 0x02, 0x00, 0x01, 0x12, 0x04, 0xa3, 0x01, 0x08, 0x0b,
+    0x0a, 0x0d, 0x0a, 0x05, 0x04, 0x16, 0x02, 0x00, 0x03, 0x12, 0x04, 0xa3, 0x01, 0x0e, 0x0f, 0x0a,
+    0x0c, 0x0a, 0x04, 0x04, 0x16, 0x02, 0x01, 0x12, 0x04, 0xa4, 0x01, 0x02, 0x12, 0x0a, 0x0d, 0x0a,
+    0x05, 0x04, 0x16, 0x02, 0x01, 0x05, 0x12, 0x04, 0xa4, 0x01, 0x02, 0x07, 0x0a, 0x0d, 0x0a, 0x05,
+    0x04, 0x16, 0x02, 0x01, 0x01, 0x12, 0x04, 0xa4, 0x01, 0x08, 0x0d, 0x0a, 0x0d, 0x0a, 0x05, 0x04,
+    0x16, 0x02, 0x01, 0x03, 0x12, 0x04, 0xa4, 0x01, 0x10, 0x11, 0x62, 0x06, 0x70, 0x72, 0x6f, 0x74,
+    0x6f, 0x33,
 ];
+include!("sync.tonic.rs");
```

Comment:
🤔 .... Does this mean that the whole client implementation is included here?

Reply:
not the client implementation just the client types.

Reply:
I don't think so. If you look at that file, it's a full-fledged gRPC client implementation for the `sync`-service.